src/bench/txreconciliation.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2020-2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <bench/bench.h>
+
+#include <net.h>
```

Comment:
In "add bench for ShouldFanoutTo" 3f59bf83a41b7787f9c43c277b5e62f327a72c3e: `net.h` include seems unnecessary.

Reply:
Removed